src/esp/physics/objectManagers/ArticulatedObjectManager.h

```diff
@@ -121,6 +129,7 @@ class ArticulatedObjectManager
       float massScale = 1.0,
       bool forceReload = false,
       bool maintainLinkOrder = false,
+      bool intertiaFromURDF = false,
```

Comment:
Would it be valuable to specify this flag in the content (via attributes) instead of the parameter?